### ISSUE TITLE
Ignoring "NLogRuntimeException: allLayouts is null. Call base.InitializeTarget()", will still disables logging

### ISSUE THREAD

**Issue by 304NotModified** [NLog/NLog#1384]
With wrong targets, `base.InitializeTarget()` isn't called and this [will lead to an exception.](https://github.com/NLog/NLog/blob/23d51b93d52b752221ca9702fdf160da488ca05f/src/NLog/Targets/Target.cs#L315)

We can ignore that exception, but still nothing gets logged due to the [initializeException field](https://github.com/NLog/NLog/blob/23d51b93d52b752221ca9702fdf160da488ca05f/src/NLog/Targets/Target.cs#L54)

**Type** (choose one):

- Bug

**NLog version**: 4.3.0

**Platform**: All

**Current NLog config** (xml or C#, if relevant)

- What is the current result? if ignore error, logging still doesn't work.
- What is the expected result?  if ignore error, just continue (note: but can crash later)

- Please post full exception details (message, stacktrace, inner exceptions)
```
NLog.NLogRuntimeException: NLog.Windows.Forms.RichTextBoxTarget.allLayouts is null. Call base.InitializeTarget() in NLog.Windows.Forms.RichTextBoxTarget
```
- Are there any work arrounds? **no**
**- Is there a version in which it did worked?  4.2.3

**Comment by 304NotModified** [comment]
resolution: move to warning, don't set initializeException field

**Comment by 304NotModified** [comment]
working on this

**Comment by 304NotModified** [comment]
This has been fixed in NLog 4.3.1 and is now online: https://www.nuget.org/packages/NLog/4.3.1
